sdks/java/extensions/sketching/build.gradle

```diff
@@ -26,15 +27,14 @@ def streamlib_version = "2.9.5"
 def tdigest_version = "3.2"
 
 dependencies {
+  compile library.java.slf4j_api
```

Comment:
I'm pretty sure this one is actually safe to remove. `slf4j` is used for logging, and this project doesn't contain any loggers. If anyone wants to add logging to this project, it's easy to add this dependency back in.

Reply:
@ibzib I'm curious how to determine whether this project contains loggers, and why removing this particular dependency entirely seems safer than removing others (where we are adding permitUnusedDeclared).

Reply:
Removed.